funnel/forms/project.py

```diff
@@ -167,6 +167,7 @@ class ProjectLivestreamForm(forms.Form):
                             'y2u.be',
                             'www.vimeo.com',
                             'vimeo.com',
+                            'player.vimeo.com',
```

Comment:
This doesn't match the regex, which will also `player.youtube.com` even though that is invalid.